LayoutTests/imported/w3c/web-platform-tests/web-animations/animation-model/animation-types/accumulation-per-property-001-expected.txt

```diff
@@ -278,8 +278,6 @@ PASS flood-color supports animating as color of hsla()
 PASS flood-opacity (type: opacity) has testAccumulation function
 PASS flood-opacity: [0, 1] number
 PASS flood-opacity: [0, 1] number (clamped)
-PASS font-stretch (type: percentage) has testAccumulation function
-PASS font-stretch: percentage
```

Comment:
So with the aliasing, I suppose this should be kept, but we should be adding an entry for `font-width` _as well_.